src/CGCOMP.CPP

```diff
@@ -1441,6 +1618,15 @@ void IZXRAT::iz_GetExteriorConditions(
 		iz_cpr * windPresV	// pressure coeff * velocity pressure
 		- iz_hz * iz_rho2;	// plus stack
 }	// IZXRAT::iz_GetExteriorConditions
+//-----------------------------------------------------------------------------
+void IZXRAT::iz_GetDOASConditions()
+// set source conditions to DOAS supply
+{
+	DOAS* pDOAS = doasR.GetAt(iz_doas);
+	iz_air2 = static_cast<AIRSTATE>(pDOAS->oa_supAF);
+	iz_rho2 = iz_air2.as_RhoMoist();
```

Comment:
Following on previous comment: The density used for the Air Net is at the supply air state, where the density used for fan energy calculations is outdoor air density.